sql/hive/src/test/scala/org/apache/spark/sql/hive/client/VersionsSuite.scala

```diff
@@ -218,6 +218,12 @@ class VersionsSuite extends SparkFunSuite with Logging {
         holdDDLTime = false)
     }
 
+    test(s"$version: tableExists") {
+      // No exception should be thrown
```

Comment:
https://github.com/apache/hive/blob/ecab0d072b50a8d85dca6e850e47425d96c1ac09/ql/src/java/org/apache/hadoop/hive/ql/metadata/Hive.java#L1216

We can let Hive return an exception. However, we set the flag to [false](https://github.com/gatorsmile/spark/blob/ce35aa01894e83761226db38d3ca0ce720de5284/sql/hive/src/main/scala/org/apache/spark/sql/hive/client/HiveClientImpl.scala#L351).